### ISSUE TITLE
vACDM

### ISSUE THREAD

**Issue by Adrian2k** [Vatsim-Scandinavia/ENOR-Sectorfile#262]
Krister showed me this plugin a while ago, but there is not a lot of documentation associated with it.
https://vacdm.net/
https://github.com/vACDM
https://discord.gg/uspkrZzrWv

It is a CDM plugin with a server backend, which also lets pilots use a website to edit their TOBT + monitor their TSAT.

It could be something to investigate more into.
We need to evaluate potential differences from the current CDM plugin and how the workflow is compared to the current plugin.

There has also been some short discussions regarding it in the ECFMP discord.

**Comment by marud94** [comment]
Currently in VERY alpha state. Only gives TSATs and takes MDI/ADI into account for CTOT. Nothing else. Must be setup by the developer, no means to change anything ourselves for now. Documentation is lacking or flawed. 

Pros: VDGS display showing TSAT with countdown, expected runway and SID.
Cons: For now must run ES in Easy correlation due to plugin code. They are working to fix it.

I still see this as a big winner if the developers get it right. 

Example from website:
"As soon as the TSAT is displayed in dark green, startup can be issued. If the pilot calls in before that and it is still shown in light green, the TOBT can be set to now and the TSAT will be recalculated. That new TSAT will not be later than the previous one, but might be earlier if there is a gap in the sequence."

Currently in talks with the developer, so we'll see.

**Comment by meltinglava** [comment]
Just flew from EDDL. The plugin did not use the filed TSAT so all pilots has to put in a number. Even if they are not a thing stopping them to depart.

**Comment by marud94** [comment]
Current status from DEV:

"We are currently working on further documentation.

For the time, vACDM is able to recognise bookings from the bmac slots tool.
Just insert the respective api-URL when start the vACDM application (docker
container).

Runway configuration is a thing we have in the pipeline for future
development (v2). For now, vACDM is only able to take dependent departure
runways into account.

Airport config is nearly the same like „the other plug-in“ and you are
fully flexible to administrate local configurations if you created your own
vACDM container"

"It’s simply setting up a docker image and fill in the correct ENV values.
Then download the latest plugin dll and set the url in the config file.
Maybe I’ll get some docu done until the end of the week."

ons. 13. sep. 2023, 14:40 skrev Magnus R ***@***.***>:

**Comment by blt950** [comment]
> For the time, vACDM is able to recognise bookings from the bmac slots tool

So this only works with booked slots for now? Or do I misunderstand?
BMAC = Our booking page for FSS etc.

**Comment by marud94** [comment]
It works like the other CDM plugin we currently use, but if you want event
slots for example, then that is the only way to implement the slots.

It has limited features as of now as it only works with single or dependent
runways and only recognizes MDIs from ECFMP. They are currently working on
expanding the features however.

lør. 23. sep. 2023, 14:48 skrev Daniel ***@***.***>:

**Comment by Adrian2k** [comment]
Is this plugin easier to handle for controllers than our current plugin? Does it require any input from the controllers?

Would it be an idea as a pilot project to test it on ENBR only?

Suggestion is based on the experience we have with using our current CDM plugin during our weekly events has been somewhat a mixed experence so far.

Also believe it would be very nice for the pilots to actually see their CTOTs.

Does this plugin allow us to issue delay for everyone if we start having a backlog of aircrafts at the holding point?

What do we need to contribute with in regards of web/server?

**Comment by marud94** [comment]
Testing version / help with initial setup requested from dev. Unsure of differences handling the day to day working of the plugin until tested.

I agree ENBR only for testing until they add option for parallell ops (only dependent runways supported as of now). Intital testing only for sectorfile dev / Staff and open for more when we are a bit more confident in the system.

Should be able to issue delay in some form, probably easier than the current way in the other cdm plugin. 

It needs to run a docker container and a mongo db ref https://vacdm.net/docs/developer/installation

Could run on adbj.no/vacdm for internal testing, but public testing in the sectorfile should be on vacdm.vatsim-scandinavia.org

**Comment by marud94** [comment]
Todo:

- [ ] Update to v 1.2.1 
- [ ] Resume ENBR testing on sweatbox
- [ ] Start ENBR testing live on VATSIM

**Comment by marud94** [comment]
Due to issues with getting data from ES the plugin can't project taxizone coordinates correctly on S-mode. Fix for now is to use Easy mode when using vACDM

**Comment by meltinglava** [comment]
> Due to issues with getting data from ES the plugin can't project taxizone coordinates correctly on S-mode. Fix for now is to use Easy mode when using vACDM

Think it is this setting which makes it not work:
![image](https://github.com/user-attachments/assets/b2eb894d-01ad-46a8-8356-d4a3a5fac1a5)
VStrips had the same issue.

**Comment by marud94** [comment]
Dev has implemented a fix, will be released soon enough

**Comment by marud94** [comment]
Agreed. I’ll leave it open until I have tested the function in CDM some more, but will close it if satisfied.

**Comment by thor** [comment]
I will mention that the Danes are trialling a french fork of the vACDM
project which fixes a few bugs here and there as an FWIW FYI.
